### PR TITLE
Change Galea channels 7 and 8 to EMG

### PR DESCRIPTION
**Pull request by isaacplotkin**
The team wants to change channels 7 and 8 to EMG as the default setting for Galea Beta.

### CODE REVIEW

**Review by Andrey1994 — CHANGES_REQUESTED** [review]
need to do the same for serial board and maybe need ti change gains in gain tracker and in code for parsing

**Comment by isaacplotkin** [comment]
beta_default_hardware_settings
Updated gain tracker to reflect new defaults in Galea GUI.

File: src/board_controller/openbci/inc/openbci_gain_tracker.h
Comment (Andrey1994) [inline src/board_controller/openbci/inc/openbci_gain_tracker.h:1]: you forgot to change default values in constructor

**Review by Andrey1994 — CHANGES_REQUESTED** [review]

**Comment by retiutut** [comment]
Confirming these are correct changes. Thanks @isaacplotkin

**Review by Andrey1994 — APPROVED** [review]

**Pull request merged**